x-pack/platform/plugins/shared/cases/public/containers/api.ts

```diff
@@ -428,6 +428,75 @@ export const patchComment = async ({
   return convertCaseToCamelCase(decodeCaseResponse(response));
 };
 
+export const patchAlertComment = async ({
+  caseId,
+  commentUpdate,
+  signal,
+}: {
+  caseId: string;
+  commentUpdate: AlertAttachment;
+  signal?: AbortSignal;
+}): Promise<AlertAttachment> => {
+  const response = await KibanaServices.get().http.fetch<AlertAttachment>(
+    getCaseCommentsUrl(caseId),
+    {
+      method: 'PATCH',
+      body: JSON.stringify({
+        ...commentUpdate,
+      }),
+      signal,
+    }
+  );
+
+  return response;
+};
+
+export const deleteAlertComment = async ({
```

Comment:
`removeAlertsFromComment` might be a better name. With `alertId` defaulting to `string[]`